<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthForm</class>
 <widget class="QDialog" name="AuthForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>259</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>447</width>
    <height>259</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>447</width>
    <height>259</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Авnоризация</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_auth">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>158</width>
        <height>74</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Войти</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_auth_login">
         <property name="text">
          <string>Логин</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_auth_login"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_auth_password">
         <property name="text">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_auth_password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_register">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>429</width>
        <height>142</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Новый пользователь</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_register_email">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_register_email">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>32767</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_register_password">
         <property name="text">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_register_password"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_register_login">
         <property name="text">
          <string>Логин</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_register_login"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_register_rate">
         <property name="text">
          <string>Валюта счёта</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_register_rate">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtCurrent</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AuthForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AuthForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
